aspnetcore/mvc/razor-pages/index.md

```diff
@@ -5,61 +5,59 @@ description: Overview of Razor Pages in ASP.NET Core
 keywords: ASP.NET Core, Razor Pages
 ms.author: riande
 manager: wpickett
-ms.date: 06/28/2017
+ms.date: 07/28/2017
 ms.topic: get-started-article
 ms.assetid: 30e4104c-0124-477b-86b3-beb7b34ed3f6
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: mvc/razor-pages/index
 ---
-## Introduction to Razor Pages in ASP.NET Core
+# Introduction to Razor Pages in ASP.NET Core
 
-By [Ryan Nowak](https://github.com/rynowak) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Ryan Nowak](https://github.com/rynowak) 
 
 Razor Pages is a new feature of ASP.NET Core MVC that makes coding page-focused scenarios easier and more productive.
 
 Razor Pages requires ASP.NET Core 2.0.0 or later. Tooling support for Razor Pages ships in Visual Studio 2017 Update 3 or later.
 
-## Getting started
+[Download or view sample code](https://github.com/Rick-Anderson/razor-page-intro).
 
-Razor Pages is on by default in MVC. If you are using a typical *Startup.cs* like the following code, Razor Pages is enabled:
+<a name="prerequisites"></a>
 
-```c#
-public class Startup
-{
-    public void ConfigureServices(IServiceCollection services)
-    {
-        services.AddMvc(); // Includes support for pages and controllers.
-    }
+## ASP.NET Core 2.0 preview prerequisites
 
-    public void Configure(IApplicationBuilder app)
-    {
-        app.UseMvc();
-    }
-}
-```
+ASP.NET Core 2.0 preview  is included in .NET Core 2.0 preview. [Download .NET Core 2.0 Preview 2](https://www.microsoft.com/net/core/preview) 
+
+Optional: [Visual Studio 2017 Preview](https://www.visualstudio.com/vs/preview/)
+
+## Razor Pages
+
+If you are using a typical *Startup.cs* like the following code, Razor Pages is enabled:
+
+[!code-cs[main](index/sample/RazorPagesIntro/Startup.cs?name=Startup)]
 
-All the new Razor Pages types and features are included in the `Microsoft.AspNetCore.Mvc.RazorPages` assembly. If you are referencing the `Microsoft.AspNetCore.Mvc` package, then a reference to the Razor Pages assembly is already included.
+All the Razor Pages types and features are included in the `Microsoft.AspNetCore.Mvc.RazorPages` assembly. If you are referencing the `Microsoft.AspNetCore.Mvc` package, a reference to the Razor Pages assembly is included.
```

Comment:
I assume Razor Pages is also included in the `Microsoft.AspNetCore.All` metapackage used by ASP.NET Core 2.0 apps targeting .NET Core 2.0. Is it worth mentioning that here?